named_type_impl.hpp

```diff
@@ -80,7 +80,7 @@ class FLUENT_EBCO NamedType : public Skills<NamedType<T, Parameter, Skills...>>.
     };
 
 private:
-    T value_;
+    T value_{};
```

Comment:
I'm not sure this is the responsibility of NamedType. If the underlying type doesn't have value init, then maybe NamedType should have the same behaviour.
I'll take care of removing this.

Reply:
I agree. I'm also not sure if this would even work the way I thought it would a the time. If `T` is default constructable then theres no need to explicitly default construct it and if `T` is not default constructable I don't think the compiler can magically generate a default constructor for that type